<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Efficiency</class>
 <widget class="QWidget" name="Efficiency">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>975</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QSpinBox" name="spinBox_efficiency_hvmin">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>490</y>
     <width>61</width>
     <height>27</height>
    </rect>
   </property>
   <property name="maximum">
    <number>3000</number>
   </property>
   <property name="singleStep">
    <number>20</number>
   </property>
   <property name="value">
    <number>800</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_efficiency_hvmax">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>530</y>
     <width>61</width>
     <height>27</height>
    </rect>
   </property>
   <property name="maximum">
    <number>3000</number>
   </property>
   <property name="singleStep">
    <number>20</number>
   </property>
   <property name="value">
    <number>1200</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_efficiency_step">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>570</y>
     <width>61</width>
     <height>27</height>
    </rect>
   </property>
   <property name="maximum">
    <number>500</number>
   </property>
   <property name="singleStep">
    <number>5</number>
   </property>
   <property name="value">
    <number>25</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_efficienc_hvmin">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>495</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>HV min</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_efficiency_hvmax">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>535</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>HV max</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_efficiency_step">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>575</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Step</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_efficiency_channel">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>450</y>
     <width>61</width>
     <height>27</height>
    </rect>
   </property>
   <property name="maximum">
    <number>5</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_efficiency_channel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>455</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Channel</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>241</width>
     <height>301</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLabel" name="label_efficiency_1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Rotary Switches</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_hv_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>VMAX (V)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_hv_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>IMAX (ÂµA)</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdNumber_efficiency_hv_vmax">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>100</y>
      <width>61</width>
      <height>23</height>
     </rect>
    </property>
    <property name="digitCount">
     <number>6</number>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdNumber_efficiency_hv_imax">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>130</y>
      <width>61</width>
      <height>23</height>
     </rect>
    </property>
    <property name="digitCount">
     <number>6</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_efficiency_hv_rotary_switches">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>35</y>
      <width>61</width>
      <height>24</height>
     </rect>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhDigitsOnly|Qt::ImhUppercaseOnly</set>
    </property>
    <property name="prefix">
     <string/>
    </property>
    <property name="maximum">
     <number>65535</number>
    </property>
    <property name="value">
     <number>52224</number>
    </property>
    <property name="displayIntegerBase">
     <number>16</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_hv_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Board model</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_efficiency_board_model">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>70</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>UNKNOWN</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_hv_1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>171</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>HV Board Panel</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_scaler_1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>181</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Scaler Board Panel</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_efficiency_scaler_rotary_switches">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>205</y>
      <width>61</width>
      <height>24</height>
     </rect>
    </property>
    <property name="maximum">
     <number>65535</number>
    </property>
    <property name="value">
     <number>13056</number>
    </property>
    <property name="displayIntegerBase">
     <number>16</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_scaler_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>210</y>
      <width>121</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Rotary Switches</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_efficiency_module_type">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>235</y>
      <width>111</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>240</y>
      <width>101</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Module Type</string>
    </property>
   </widget>
  </widget>
  <widget class="QCustomPlot" name="qcp_efficiency" native="true">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>100</y>
     <width>631</width>
     <height>521</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_efficiency_start">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>10</y>
     <width>71</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Start</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_efficiency_stop">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>10</y>
     <width>71</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Stop</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_efficiency_clear">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>10</y>
     <width>71</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Clear</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_efficiency_scaler_width">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>330</y>
     <width>61</width>
     <height>27</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>254</number>
   </property>
   <property name="value">
    <number>241</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>335</y>
     <width>41</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>width</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_efficiency_scaler_unit">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>330</y>
     <width>85</width>
     <height>27</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>3</number>
   </property>
   <item>
    <property name="text">
     <string>25 ns</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1600 ns</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>410 Âµs</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>104 ms</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>300</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Gate configuration</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>420</y>
     <width>231</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Efficiency configuration</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>370</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Repeat</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_efficiency_scaler_repeat">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>365</y>
     <width>61</width>
     <height>27</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>20</number>
   </property>
   <property name="singleStep">
    <number>1</number>
   </property>
   <property name="value">
    <number>4</number>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressBar_efficiency">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>50</y>
     <width>631</width>
     <height>23</height>
    </rect>
   </property>
   <property name="value">
    <number>0</number>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_efficiency_file_name">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>625</y>
     <width>511</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_efficiency_file_name">
   <property name="geometry">
    <rect>
     <x>860</x>
     <y>625</y>
     <width>71</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Change</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>630</y>
     <width>31</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>File</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
